Form-Controls/styles.css

```diff
@@ -0,0 +1,47 @@
+
+h1{
+    text-align: center;
+    color: #354f52;
+}
+
+h2{ 
+    text-align: center;
+    color: #354f52;
+    font-size: 12px;
+}
+
+div{
+    margin-bottom: 20px;
+}
+
+.product-form{
+    background-color: #588157;
```

Comment:
This is interesting, I think.

This background colour with black text (default) has a contrast percentage of 32% using the [perceptual colour algorithm](https://web.dev/color-and-contrast-accessibility/?utm_source=devtools)
<img width="249" alt="Screenshot 2023-01-15 at 10 11 22" src="https://user-images.githubusercontent.com/42203406/212535266-df2bafe2-9298-4732-8e70-e9643f6e798b.png">

But it gets full marks on the W3C contrast check
<img width="347" alt="Screenshot 2023-01-15 at 10 27 07" src="https://user-images.githubusercontent.com/42203406/212535529-dc790f60-cab4-484e-a2f3-d274ea749035.png">

I checked this because I find this hard to read myself and I had already run Lighthouse on your code and noted its 100% pass rate ( 🏅 ) . This is interesting to think about more. Why is this text hard to read? How could you catch this problem in testing?